test/e2e/framework/framework.go

```diff
@@ -401,7 +400,7 @@ func (f *Framework) CreateNamespace(baseName string, labels map[string]string) (
 	f.AddNamespacesToDelete(ns)
 
 	if err == nil && !f.SkipPrivilegedPSPBinding {
-		e2epsp.CreatePrivilegedPSPBinding(f.ClientSet, ns.Name)
+		CreatePrivilegedPSPBinding(f.ClientSet, ns.Name)
```

Comment:
This part is a really hard point to remove dependency to e2epsp subpackage from the core framework.
I tried to remove this part, but I found it is so difficult and gave up.

Reply:
The CI tests passed. I think move psp.go under framework is resonable and harmless. The intention is that the framework does not use subpackages and all subpackages use the same logging framework. There are only two functions called by `framework` and `auth`, the function call from `framework` is hard to remove as you say , why don't we make psp.go as a part of framework.